src/lib.rs

```diff
@@ -1,24 +1,24 @@
-/// ewasm_api is a library used to interface with Ethereum's EEI in Ewasm, a set of enhancements to
-/// the Ethereum smart contract platform.
-/// ewasm_api exposes both a set of unsafe "native" functions representing the actual EEI
-/// functions, and a set of safe wrappers around them. It is recommended not to use the native
-/// functions as they do not perform bounds-checking.
-///
-/// To use ewasm_api, simply include it as a dependency in your project.
-///
-/// # Examples
-/// ```ignore
-/// extern crate ewasm_api;
-///
-/// use ewasm_api::{Hash, block_hash, finish_data};
-///
-/// #[cfg(target_arch = "wasm32")]
-/// #[no_mangle]
-/// pub extern "C" fn main() {
-///     let a: Hash = block_hash(1);
-///     finish_data(&a.bytes);
-/// }
-/// ```
+//! ewasm_api is a library used to interface with Ethereum's EEI in Ewasm, a set of enhancements to
+//! the Ethereum smart contract platform.
+//! ewasm_api exposes both a set of unsafe "native" functions representing the actual EEI
+//! functions, and a set of safe wrappers around them. It is recommended not to use the native
+//! functions as they do not perform bounds-checking.
+//!
+//! To use ewasm_api, simply include it as a dependency in your project.
+//!
```

Comment:
Should mention the existence of features (no_std, wee_alloc/qimalloc, debug, experimental).